scripts/_common.sh

```diff
@@ -95,7 +95,8 @@ init_composer() {
     || ynh_die "Unable to install Composer"
 
   # update dependencies to create composer.lock
-  exec_composer "$DESTDIR" install  --no-dev --prefer-dist --no-scripts  \
+  exec_composer "$DESTDIR" update symfony/flex --no-plugins --no-scripts
```

Comment:
this line should only be executed for updates, not for new installs

Reply:
I have added "update symfony/flex --no-plugins" only for update. Please see 
https://github.com/YunoHost-Apps/kimai2_ynh/pull/3/commits/436fe9d924e295bfdd262c3146585b58104d7743#diff-dc2801cb46a5d60e62234cff98084498R119